_posts/en/newsletters/2020-06-03-newsletter.md

```diff
@@ -0,0 +1,299 @@
+  state of LN backups. He discussed the approaches of other
+  implementations such as Eclair and LND before explaining why
+  C-Lightning is using a synchronous database log plugin. Later, he
+  described why LN backups are more complex than onchain backups, the
+  prospects of getting [SIGHASH_NOINPUT][topic sighash_noinput] or
+  `SIGHASH_ANYPREVOUT` into Bitcoin to enable [eltoo-based][topic eltoo]
+  payment channels, and the current modularity of the LN protocol.
+  ([transcript][decker xs], [video][decker vid])
+
+- **Payjoin/P2EP:** Adam Gibson led a discussion at London BitDevs about
+  [payjoin][topic payjoin], including the recent implementation of it by
+  BTCPay Server.  Payjoin allows both the sender and receiver of a
+  payment to coordinate to increase their privacy by both contributing
+  inputs to the transaction, breaking the [common wallet ownership
+  assumption][] and subset sum analysis. Gibson went through the history
+  of the concept and described the existing implementations of payjoin
+  in JoinMarket and Samourai before examining details of the recent
+  BTCPay Server implementation. He ended by outlining several different
+  ways a wallet can be fingerprinted, such as the number of signatures
+  required, what timelocks are used, and whether the opt-in
+  Replace-By-Fee ([RBF][topic rbf]) flag is set.  ([transcript][gibson
+  xs], [video][gibson vid])
+
+- **LSAT---Your Ticket Aboard The Lightning Native Web:** Oliver Gugger
+  presented on the Lightning Service Authentication Token (LSAT) at
+  Reckless VR in virtual reality. LSAT is a proposed protocol
+  specification combining HTTP, macaroons, and Lightning. It's designed
+  to fulfill the purpose of the HTTP 402 Payment Required response code.
+  Gugger described the authentication flow and the role of macaroons as
+  pseudonymous user authentication. The question and answer session
+  focused on use cases and the benefits of using LSAT, such as enhanced
+  user privacy and improved sign up experience.
+  ([transcript](https://diyhpl.us/wiki/transcripts/vr-bitcoin/2020-05-16-oliver-gugger-lsat/),
+  [video](https://www.youtube.com/watch?v=IW08RJUpzw0))
+
+- **Sydney meetup discussion:** A number of Bitcoin and LN developers
+  joined this Sydney meetup to discuss topics including: the scalability
+  issues of onboarding millions of LN clients, Rust code integration
+  into Bitcoin Core, dual funding in C-Lightning, and modern soft fork
+  activation. The history of Linux kernel development and segwit
+  activation were explored for insights into when or whether the Rust
+  language should be introduced to Bitcoin Core and the exact mechanism
+  that should be used to activate proposed soft forks such as
+  [taproot][topic taproot] in future. The transcript was anonymized to
+  encourage open discussion.  ([transcript][sydney xs])
+
+- **Revault—A Multiparty Vault Architecture:** Kevin Loaec and Antoine
```

Comment:
This may be correct and only just me, but I initially wondered what Revault—A was.

Maybe s/Revault—A Multiparty/Revault — A Multiparty/